dist/datasource.ts

```diff
@@ -329,20 +329,95 @@ export default class GoogleStackdriverDatasource {
     });
   }
 
-  getMetricLabel(aliasPattern, series) {
-    let aliasRegex = /\{\{(.+?)\}\}/g;
+  filterSeries(target, response) {
+    if (!_.has(target, 'seriesFilter') ||
+        target.seriesFilter.mode === 'NONE' ||
+        target.seriesFilter.type === 'NONE' ||
+        target.seriesFilter.param === '') {
+      return response;
+    }
+
+    let param = _.toNumber(target.seriesFilter.param);
+    if (_.isNaN(param)) return response;
```

Comment:
is there a better way to check if a string is a valid number?

Reply:
It is popular to import the underscore lib this way. I'd leave it, otherwise I can never remember is this is a built-in or library method